CHANGELOG.md

```diff
@@ -9,17 +9,19 @@ Changelog
     - `Epoch.blocksCount`
 - `Cardano.networkName`
 - `Transaction.size`
-## Fixes
-- The aggregated and known very large numbers are now typed as String. Cardano JS has utilities to work with these return values, currently limited to currency conversion.
-- `cardano` query return value is accurately defined to always return an object of type `Cardano` 
-- `Transaction.fee` was a String, now a `BigInt`
 - Allow explicit ordering of `Transaction.input` and `Transaction.outputs` by their natural `index`
+- `Cardano` now matches the postgres view, and is an improvement over the previous version which performed two queries.
+
+## Breaking changes
+- Transaction and Block IDs are now labelled as `hash`, aligning with the domain terminology.
+- `Block.merkelRootHash` -> `Block.merkelRoot`
+- The aggregated and known very large numbers are now typed as String. Cardano JS has utilities to work with these return values, currently limited to currency conversion.
```

Comment:
I still find this one weird. Likely, for numbers that are this big (> 2^53), we probably don't need an enormous precision, so it'd be fine to store them as two numbers with an exponent and a coefficient :thinking: ?

This is really just a thought.

Reply:
I agree, however with the current design the `cardano-graphql` server is just routing requests and responses to and from Hasura to keep the complexity low. I'd like to revisit this before the next major release